teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1918 @@
+{
```

Comment:
<blockquote>This issue can be solved leveraging similarities and differences between proteins.</blockquote><p>-&gt; rather 'partially solved'?</p><p> </p><p>And maybe add a general link to adenosine receptors?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>